pkgs/development/python-modules/jaxtyping/default.nix

```
@@ -30,15 +30,15 @@ let
       owner = "google";
       repo = "jaxtyping";
       rev = "refs/tags/v${version}";
-      hash = "sha256-xDFrgPecUIfCACg/xkMQ8G1+6hNiUUDg9eCZKNpNfzs=";
+      hash = "sha256-kuGFzp8sDLq6J/qq8ap3lD3n1pABHurXcbRUtDQyWwE=";
     };
 
-    nativeBuildInputs = [
+    build-system = [
       hatchling
       pythonRelaxDepsHook
     ];
 
-    propagatedBuildInputs = [
+    dependencies = [
       numpy
       typeguard
       typing-extensions
```

```suggestion
      typeguard
```
https://github.com/patrick-kidger/jaxtyping/compare/v0.2.28...v0.2.31#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R26

---

Indeed I missed that...